src/test/java/net/eventsexpress/app/tests/PrivacyPageTest.java

```diff
@@ -0,0 +1,24 @@
+package net.eventsexpress.app.tests;
+
+import net.eventsexpress.app.pages.Footer;
+import net.eventsexpress.app.pages.footerpages.PrivacyPage;
+import org.testng.annotations.Test;
+
+public class PrivacyPageTest extends BaseTest {
+    @Test
+    public void testThatUserIsRedirectedToPrivacyPolicyPage() {
+        new Footer()
+                .navigateToPrivacyPage();
+        new PrivacyPage()
+                .assertPrivacyHeadingIsDisplayed();
+    }
+
+}
+
+
+
+
+
+
+
+
```

Comment:
remove extra lines, leave only one at the bottom

Reply:
extra have been deleted